packages/shared-core/src/constants/index.ts

```diff
@@ -179,3 +179,5 @@ export enum BpmStatusValue {
   VERIFYING_EMAIL = "verifying_email",
   COMPLETED = "completed",
 }
+
+export const DEFAULT_BB_DATASOURCE_ID = "datasource_internal_bb_default"
```

Comment:
Is this duplicated, I believe there is somewhere in backend-core or server that defines this as well?

Reply:
Yea I wanted to add it in to frontend-core as it didn't exist there, so thought it made sense to add to shared-core for that purpose. I can't import from backend-core or server in shared-core can I?

Reply:
You can remove it from everywhere else though and then import it from shared-core to be exported from backend-core - in `backend-core/src/constants/db.ts` you can do `export { DEFAULT_BB_DATASOURCE_ID } from "@budibase/shared-core`

Reply:
Sorted this now @mike12345567